### PR TITLE
add tableView component

### PR DESCRIPTION
**Pull request by DoruNow**
Layout:
image

### CODE REVIEW

File: src/views/HomePageView.vue [hunk 1]
```diff
@@ -58,7 +71,7 @@ import LicitatiiComponent from "@/components/LicitatiiComponent.vue";
 @Component({
   components: { LicitatiiComponent }
 })
-export default class App extends Vue {
+export default class HomePageView extends Vue {
```
Comment (DoruNow) [inline src/views/HomePageView.vue:74]: @azirbrooks This should have been caught at code review!

File: src/views/TableView.vue [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+      },
+      {
+        value: 12,
+        type: "Spades"
+      }
+    ],
+    gameScore: [1, 2],
+    sessionScore: [1, 2],
+    players: ["P1", "P2", "P3", "P4"]
+  };
```
Comment (DoruNow) [inline src/views/TableView.vue:86]: This is the data structure needed for the table. Everything has to have defaults except for `cards[x].type`

Comment (azirbrooks) [reply src/views/TableView.vue:86]: @DoruNow  How do you know who played each card if you removed playedBy property?

File: src/views/HomePageView.vue (hunk 1 above)
Comment (azirbrooks) [reply src/views/HomePageView.vue:74]: @DoruNow Failing to catch that is related to my limited experience with Vue

**Pull request merged**